recipe/generator_test.go

```diff
@@ -14,41 +14,74 @@ func TestFromTemplate(t *testing.T) {
 	t.Run("should output recipe files using template to output directory", func(t *testing.T) {
 		templatePath := "./testdata/generator/template.yaml"
 		outputDir := "./testdata/generator/temp"
-		data := []recipe.TemplateData{
-			{
-				FileName: "recipe-1",
-				Data: map[string]interface{}{
-					"broker": "main-broker.com:9092",
-					"owner":  "john@example.com",
+
+		t.Run("when recipe has a name", func(t *testing.T) {
+			data := []recipe.TemplateData{
+				{
+					FileName: "recipe-one",
+					Data: map[string]interface{}{
+						"name":   "recipe-1",
```

Comment:
would it be possible to source these data from the file?
like we do in normal cli?

Reply:
Yes, did this. Thanks for the suggestion!